javascript/apps/taiga/src/app/modules/feature-user-settings/components/deleting-account-modal/deleting-account-modal.component.html

```diff
@@ -0,0 +1,184 @@
+        role="dialog"
+        aria-labelledby="deleting-account"
+        aria-modal="true">
+        <h1
+          class="title"
+          id="deleting-account">
+          {{ t('user_settings.account.delete.confirmation_modal.title') }}
+        </h1>
+        <h2 class="visually-hidden">
+          {{ t('user_settings.account.delete.confirmation_modal.description') }}
+        </h2>
+
+        <tui-scrollbar>
+          <ng-container *ngIf="vm.accountInfo?.workspaces as workspaces">
+            <section
+              class="user-admin-list"
+              *ngIf="workspaces.length">
+              <tg-ui-inline-notification status="warning">
+                <p>
+                  {{
+                    t('user_settings.account.delete.confirmation_modal.caution')
+                  }}
+                </p>
+                <p
+                  subtitle
+                  [innerHTML]="
+                    t(
+                      'user_settings.account.delete.confirmation_modal.only_admin_workspaces',
+                      {
+                        workspaces: workspaces.length
+                      }
+                    ) | safeHtml
+                  "></p>
+              </tg-ui-inline-notification>
+              <p class="description">
+                {{
+                  t(
+                    'user_settings.account.delete.confirmation_modal.workspace_description'
+                  )
+                }}
+              </p>
+              <h3
+                class="visually-hidden"
+                *ngIf="workspaces.length > 1">
+                {{
+                  t(
+                    'user_settings.account.delete.confirmation_modal.workspace_a11y_title'
+                  )
+                }}
+              </h3>
+
+              <div class="workspaces">
+                <div
+                  class="workspace"
+                  data-test="delete-user-workspace"
+                  *ngFor="let workspace of workspaces; trackBy: trackById">
+                  <tg-ui-avatar
+                    class="workspace-avatar"
+                    [color]="workspace.color"
+                    type="dark"
+                    size="m"
+                    [name]="workspace.name">
+                  </tg-ui-avatar>
+
+                  <span class="workspace-name">{{ workspace.name }}</span>
+
+                  <tg-projects-dropdown
+                    class="projects-list"
+                    [projects]="workspace.projects"></tg-projects-dropdown>
+                </div>
+              </div>
+            </section>
+          </ng-container>
+
+          <ng-container *ngIf="vm.accountInfo?.projects as projects">
+            <section
+              class="user-admin-list"
+              *ngIf="projects.length">
+              <tg-ui-inline-notification status="warning">
+                <p>
+                  {{
+                    t('user_settings.account.delete.confirmation_modal.caution')
```

Comment:
When there are 2 inline notifications the 'Caution' it should appear just in the first one.
![image](https://github.com/taigaio/taiga/assets/5970959/828b07ac-5840-4dcd-9163-b35f64b245b1)
![image](https://github.com/taigaio/taiga/assets/5970959/7fe1c9c3-545f-49c5-93c9-56ba3b23f513)